packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -87,11 +87,18 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
     // directory as the workingDirectory, to avoid hitting the limit on command
     // lengths in batch scripts if the absolute path to the project is long.
     result = await processUtils.stream(<String>[
```

Comment:
Nit: break before `<String>`? The indentation here (especially of the `],`) reads really strangely to me without that.

Reply:
Done